README.md

```diff
@@ -369,6 +388,28 @@ The `iotedgedev` module has the following commands:
 `iotedgedev project --help`
 - `--create TEXT`  Creates a new Azure IoT Edge project. Use `--create .` to create in current folder. Use `--create TEXT` to create in a subfolder.
 
+
+**azure**
+
+`iotedgedev azure --help`
+
+- `--setup`                    Reads the required Azure components
+                               configuration from your subscription. Creates
+                               new Azure resources or uses existing ones.
+                               [required]
+- `--azure-credentials TEXT TEXT`  The credentials (username password) to use to
+                               login to Azure
+- `--interactive-login`          Logs in to Azure in interactive mode
+- `--subscription TEXT`          The Azure subscription name or id to use. Leave
```

Comment:
Instead of using the default, let the user select the appropriate sub.  We don't want assets to be auto deployed to default by mistake